eslint.config.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+const neo = require('neostandard')
+
+module.exports = [
+  ...neo({
+    env: ['jest'],
+    ignores: [
+      'lib/llhttp',
+      'test/fixtures/wpt',
+      'node_modules',
+      'undici-fetch.js'
+    ],
+    ts: true
+  }),
+  {
+    rules: {
+      '@stylistic/comma-dangle': ['error', {
+        arrays: 'never',
+        objects: 'never',
+        imports: 'never',
+        exports: 'never',
+        functions: 'never'
+      }],
+      '@typescript-eslint/no-redeclare': 'off',
+      'no-lone-blocks': 'off'
```

Comment:
Was this a regression since standard? Or what prompted the addition?

Reply:
If you have 

```
{
  new Date()
}
```

you get no-lone-blocks linting error but if you do 

```
{
  const date = new Date()
}
```
You wont.

Not a regression because we didnt lint ts files before.

Removed those blocks